src/main/java/mate/controller/DeleteManufacturersController.java

```diff
@@ -0,0 +1,24 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.ManufacturerService;
+
+public class DeleteManufacturersController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private static final ManufacturerService manufacturerService = (ManufacturerService) injector
+            .getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
```

Comment:
It's better to use GET method for getting data only, please rework controller and related JSP page to use POST method for deletion